django/01-blog-websayt/blog-proyekt/04-django_admin1.md

```diff
@@ -0,0 +1,72 @@
+## Django_admin
+* 4 - kunda biz web saytlar uchun `admin` yaratishni ko'rib chiqamiz. `admin` bu web saytimizni boshqarish uchun , yangiliklarni kirtib borish, veb sayt foydalanuvchilari (users) boshqarish uchun qo'llaniladi.
+* Dastlabki qadamda biz `admin`ni homani web saytimizdan ko'rib olamaiz. web browserga homaki satimizning manzilini kiritamiz yani bu manzilni `http://127.0.0.1:8000/` hamda uning yonidan slash belgi`/` orqalim `admin` kiritamiz. Shundan so'ng bizning sahifamizda shunday ko'rinig paydo bo'ladi.
+
+<p align="center">
+    <img src="./image/img.png">
+</p>
+
+* Yuqoridagi `Django administration`ni ko'rib turibsizlar. Unda bizda oldindan admin bo'lmaganligi uchun biz uni yaratib olamiz.
+
+* 1 - qadamda `django_server`ni o'chirib olamiz.`django_project` fayl orqali biz `python manage.py createsuperuser` kiritib olamiz.Bularni kiritganizmda so'ng `terminal`da quyidagilar paydo bo'ladi.
+
+<p align="center">
+    <img src="./image/img_1.png">
+</p>
+
+* `Terminal`ning so'ngi qismiga e'tibor beradigan bo'lsak, qy=uyidagilar paydo bo'ladi `django.db.utils.OperationalError: no such table: auth_user`.Bunda biz ishga tushurmoqchi bo'lgan `superuser` ishlamadi.Chunki biz `database` yaratib olishimiz kerak bo'ladi.
+* `Database` yaratib olish uchun biz avvalo `migration` commandasi kirtib olishimiz kerak.Buning yana biz `python manage.py makemigrations` kirtib olamiz va quyidagilar paydo bo'ladi.
```

Comment:
**commandasi kirtib - commandasini kiritib**

Reply:
qy=uyidagilar >>> quyidagilar

Reply:
- .Bunda >>> . Bunda (nuqtadan keyin bo'shliq bo'lishi kerak)
- .Chunki >>> . Chunki (nuqtadan keyin  bo'shliq bo'lishi kerak)
- Xuddi shunday xatolni hammasini qaytadan to'girlab chiqish kerak.
- kirtib >>> kiritib